src/challenges/defi-challenges/simple-lending.mdx

```diff
@@ -0,0 +1,244 @@
+---
+name: Simple Lending
+index: 3
+level: 3
+website: "https://github.com/Turupawn/LendingChallengeLevelUp"
+summary: Understand the inner workings of Aave by completing the Lending challenge.
+heroImage: "/images/level-up-preview.png"
+labels: ["DeFi"]
+---
+
+import ContentCopy from "src/components/ContentCopy";
+
+# Simple Lending
+
+If you haven't completed the Token Swap challenge yet, check it out [here](https://www.levelup.xyz/challenges/simple-token-swap)!
+
+Let's take a step further and dive into another core DeFi primitive, lending. Aave is one of the top protocols, securing billions in assets. Let's learn how to earn yield by lending through smart contracts.
+
+In this challenge, you'll implement a contract that deposits into Aave pools on your behalf. You'll also be able to withdraw your supplied tokens along with the earned rewards.
+
+## Objective
+
+Your task is to:
+1. Develop your own contract with `deposit` and `withdraw` functionalities based on the provided base contract.
```

Comment:
Challenge below seems to define `stake` and `withdraw` function. Should we update `deposit` to `stake`?

Also, I notice that the majority of the challenge calls out `supply` function. Does it make more sense to just use `supply` for rest of the challenge?

Reply:
The thing is the Aave functions are called `deposit` and `withdraw`, and I named the challenge functions `stake` and `withdraw` so they are different. But yeah that sounds confusing.

So I think we should rename the challenges functions to `proxyDeposit` and `proxyWithdraw` to make a clear distinction. Other option could be `stake` and `unstake`. What do you think?

Reply:
That makes sense! Personally, I think `stake` and `unstake` provides more clarity. I think then just updating the `deposit` and `withdraw` pair for the rest of this challenge will be sufficient, and we can ship it!